### PR TITLE
Optaplanner : upgrade to 8.x extension #2243

### PR DESCRIPTION
**Pull request by zbendhiba**
[ ] An issue should be filed for the change unless this is a trivial change (fixing a typo or similar). One issue should ideally be fixed by not more than one commit and the other way round, each commit should fix just one issue, without pulling in other changes.
[ ] Each commit in the pull request should have a meaningful and properly spelled subject line and body. Copying the title of the associated issue is typically enough. Please include the issue number in the commit message prefixed by #.
[ ] The pull request description should explain what the pull request does, how, and why. If the info is available in the associated issue or some other external document, a link is enough.
[ ] Phrases like Fix #<issueNumber> or Fixes #<issueNumber> will auto-close the named issue upon merging the pull request. Using them is typically a good idea.
[ ] Please run mvn process-resources -Pformat (and amend the changes if necessary) before sending the pull request.
[ ] Contributor guide is your good friend: https://camel.apache.org/camel-quarkus/latest/contributor-guide.html

### CODE REVIEW

**Review by oscerd — APPROVED** [review]

File: poms/bom/pom.xml [hunk 1]
```diff
@@ -5998,29 +5998,29 @@
...
-                <groupId>org.kie.soup</groupId>
-                <artifactId>kie-soup-project-datamodel-commons</artifactId>
-                <version>${optaplanner.version}</version>
-            </dependency>
-            <dependency>
-                <groupId>org.mvel</groupId>
-                <artifactId>mvel2</artifactId>
-                <version>${mvel2.version}</version>
-            </dependency>
+             <dependency>
```
Comment (jamesnetherton) [inline poms/bom/pom.xml:6001]: Minor nit - few dependencies here got some additional whitespace added to them, so they are not aligned nicely anymore.

Comment (zbendhiba) [reply poms/bom/pom.xml]: yes!! thanks

File: poms/bom/pom.xml [hunk 2]
```diff
@@ -5999,28 +5999,28 @@
                 <version>${jsoup.version}</version>
             </dependency>
             <dependency>
-                <groupId>org.kie.soup</groupId>
-                <artifactId>kie-soup-commons</artifactId>
-                <version>${optaplanner.version}</version>
+                 <groupId>org.mvel</groupId>
```
Comment (ppalaga) [inline poms/bom/pom.xml:6002]: Nitpick: Isn't there one space too much here and on other changed lines of this file?

**Review by ppalaga — APPROVED** [review]

**Review by aldettinger — APPROVED** [review]
The build issue likely does not relate to this PR.

File: poms/bom/pom.xml (hunk 2 above)
Comment (zbendhiba) [reply poms/bom/pom.xml]: @ppalaga : It should be better right now.

**Comment by jamesnetherton** [comment]
I fixed up the `camel-master` branch issues, verified this locally & manually merged.

**Comment by zbendhiba** [comment]
> I fixed up the `camel-master` branch issues, verified this locally & manually merged.

thanks !!

**Pull request closed without merging**